src/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -98,7 +98,7 @@ private static int RunConsole(ProjectContext projectContext, CommandLineApplicat
             var commandArgs = new List<string> { projectContext.GetOutputPaths(configuration).CompilationFiles.Assembly };
             commandArgs.AddRange(app.RemainingArguments);
 
-            return Command.CreateDotNet($"{GetCommandName(testRunner)}", commandArgs, projectContext.TargetFramework)
+            return Command.Create($"dotnet-{GetCommandName(testRunner)}", commandArgs, projectContext.TargetFramework, configuration: configuration)
```

Comment:
I'm going to make an update to DotnetCommandFactory to include configuration, but this is specifically using `dotnet-command` over `dotnet command` to avoid going through a second resolution step.

The method DotnetCommandFactory calls, CreateDotNet will use the latter method.